src/runtime_src/core/pcie/linux/CMakeLists.txt

```diff
@@ -55,6 +55,8 @@ target_link_libraries(xrt_core
   rt
   dl
   uuid
+  INTERFACE
+  Boost::boost
```

Comment:
Why this change for xrt_core?   I follow the xrt_coreutil because that library is required for xrt::device which compile time depends on boost.  But xrt_core library is not linked directly (usually) to host application and even if it was, it doesn't have client requirements beyond the run-time dependencies that are installed automatically with the rpm/deb.

Reply:
We are linking both of them
```
$ ldd rt-engine/librt-engine.so 
        linux-vdso.so.1 (0x00007ffe1fdfb000)
        libxrt_core.so.2 => /opt/xilinx/xrt/lib/libxrt_core.so.2 (0x00007f8f901da000)
        libxrt_coreutil.so.2 => /opt/xilinx/xrt/lib/libxrt_coreutil.so.2 (0x00007f8f8feab000)
```

I guess I didn't know which library I should apply this to, so I chose both.

Reply:
Also, what about hardware emulation? In that case I'm also not sure which library to apply this to.

Reply:
hw_emu is same as xrt_core, it should not need the dependency.